html/index.html

```diff
@@ -186,28 +181,29 @@ <h3>Programming</h3>
                 is virtually traversing it.</p>
           </div>
         </article>
-        <div class="imgblock"></div>
-        <span> <p><right>Transcript of a conversation about program behaviour </right> </p> </span>
-        <img src="../images/dialog.png" height="80%" width="80%" alt="">
-         <div class="line"></div>
-             <article>
-               <div class='imgblock'>
-                 <img src="../images/blacksquare_89177.png" alt="">
-              <span>WHERE TO GO NEXT</span>
-              <span>
-                <p><ul>Many Cuisenaire and Gattegno resources can be found online:</p>
-                <p><li>New developments in arithmetic teaching 
+       </section>
+       <div class="imgblock"></div>
+       <span> <p><right>Transcript of a conversation about program behaviour </right> </p> </span>
+      <section><img src="../images/dialog.png" height="80%" width="80%" alt=""></section>
+        <div class="line"></div>
+             <article class="block">
+              <div class='imgblock'>
+                <img src="../images/blacksquare_89177.png" >
+             <div> 
+                <p class="caption">WHERE TO GO NEXT</p>
+                <ul>Many Cuisenaire and Gattegno resources can be found online:
+                <li>New developments in arithmetic teaching 
                 in Britain: introducing the concept of ‘set’ 
-                <a href="../html/index.html">helloworld.cc/2pYFgOw</a>.</li></p>
-                <p><li>The common sense of teaching mathematics: 
-                <a href="../html/index.html">helloworld.cc/2p9dyKO</a></li></p>
-                <p><li>Gattegno Mathematics, Book 1: 
-                <a href="../html/index.html">helloworld.cc/2qz7xZe</a>.</li></p>
-                <p><li>Virtual Rods: 
-                <a href="../html/index.html">helloworld.cc/2pI8ctc</a>.</li></p>
-                <p><li>Sample code for these exercises: 
-                <a href="../html/index.html">helloworld.cc/2pFyx9j</a>.</li></p>
-              </ul></span>
+                <a href="../html/index.html">helloworld.cc/2pYFgOw</a>.</li>
```

Comment:
what is the dot between `</a>` and `</li>`?

Reply:
are you sure that this path (in href) will redirect user to the correct link that is specified in the pdf maket ?
this applies to all of the following links in this block